app/Http/Controllers/Api/V1/CurrikiGo/LmsServicesController.php

```diff
@@ -107,9 +107,5 @@ public function saveLtiTeachersData(Request $request, GoogleClassroomRepositoryI
             $service = new SaveStudentdataService();
             $service->saveStudentData($request);
         }
-        $lmsSetting = $this->lmsSettingRepository->findByField('lti_client_id', $request->issuerClient);
-        $canvasClient = new Client($lmsSetting);
-        $saveData = new SaveTeacherData($canvasClient);
```

Comment:
why we removed this code? and testing done after removing the code?

Reply:
Its not required anymore on activity launch